bokehjs/src/coffee/core/ui_events.coffee

```diff
@@ -130,13 +130,13 @@ export class UIEvents
         if view?
           if view.model.cursor?
             cursor = view.model.cursor()
-          if active_inspectors?
```

Comment:
Pretty much everything that touches DOM may work differently in our testing environment, because jsdom, which we use to emulate DOM, is only and API and doesn't provide any functionality. Though code may still work by chance or with aid of stubs.